rfcs/checkbox-component.md

```diff
@@ -112,16 +112,6 @@ export default () => {
   * `Input` may replace original functionality of checkbox. If provided should follow the same interface and logic or may break the whole component
 * `labelPlacement: 'top' | 'right' | 'bottom' | 'left'`:
   How to position the label relative to the checkbox itself. Default is `right`
-* `onChange: func`:
```

Comment:
I don't think we should remove these handlers from the components API, since we override them and call the user's handlers within it. Could you give more details on why it's being removed?

Reply:
I am not sure  - to me it seems we do not override them at all, just passing them through as is: https://github.com/uber-web/baseui/blob/master/src/checkbox/checkbox.js#L168

What am I missing?

Reply:
Ok, we do anything with the `onChange` handler, but we do some state changes on the mouse events.

Reply:
Oh, found it - but even in that case, from a user's point of view there is no change in the API, right? We are calling anything they pass in with the original input event, right?

Reply:
That's right. But I think it introduces the issue I mentioned in my [other comment here](https://github.com/uber-web/baseui/pull/372#discussion_r223795765)

Reply:
Even if it's a pass through, I think the `onChange` handler is important enough to document since it's a required prop. The other handlers are optional and rarely used.